buzzmobile/process/bearing/bearing.py

```diff
@@ -55,7 +55,7 @@ def bearing(fix):
 
 def bearing_node():
     rospy.init_node('bearing', anonymous=True)
-    rospy.Subscriber('/fix', NavSatFix, bearing)
+    rospy.Subscriber('fix', NavSatFix, bearing)
```

Comment:
this `fix` needs to be `/fix`

If we are running our nodes in the `buzzmobile` namespace, this subscribe call is equivalent to `Subscriber('/buzzmobile/fix/', ...`. However, third-party nodes like Nmea Navsat will not be in the buzzmobile namespace, so we need to make explicit `/fix`